src/DonationForms/Actions/ConvertDonationFormBlocksToFieldsApi.php

```diff
@@ -293,9 +295,13 @@ protected function createNodeFromDonorNameBlock(BlockModel $block): Node
 
 
             if ($block->hasAttribute('showHonorific') && $block->getAttribute('showHonorific') === true) {
-                $group->getNodeByName('honorific')
-                    ->label('Title')
-                    ->options(...array_values($block->getAttribute('honorifics')));
+                $options = array_values((array)$block->getAttribute('honorifics'));
```

Comment:
Should we check for empty strings when passing the options? I noticed you can add an empty option and it will display. Although maybe it is something we can look into another time.

Reply:
nice catch! resolved f3eed612